packages/block-library/src/template-part/edit/label.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
**Note:** I was thinking about making this a HOC if we plan to use it on other blocks on the future.